tests/pipeline/resources/custom-config/defaults.yaml

```diff
@@ -1,19 +1,17 @@
-kafka-app:
-  version: 2.9.0
-  namespace: development-namespace
```

Comment:
shouldn't those values move to streams-bootstrap-v2 or are they not needed?

Reply:
true added them